apps/zotonic_core/src/models/m_rsc_update.erl

```diff
@@ -657,6 +657,18 @@ update_result({ok, NewId, {OldProps, NewProps, OldCatList, IsCatInsert}}, #rscup
         },
         Context),
 
+    % If a newly inserted resource is dependent, then schedule a check
+    % if it is connected. If not then it should be deleted.
```

Comment:
IIUC this would also be triggered for updated resources, if so I'd change the comment:
```suggestion
    % If a new or updated resource becomes dependent, then schedule a check
    % if it is connected. If not then it should be deleted.
```